app/templates/events/import.hbs

```diff
@@ -1 +1,2 @@
-import data
+{{events/event-import-section}}
```

Comment:
Umm.. No. I meant... Add a line break and push the content down.(by pushing down, I mean moving the content down. It's too near to the tab now)